wigwag/system/bin/info

```diff
@@ -370,21 +420,26 @@ _memperf(){
 }
 
 performance(){
-	_placeTitle "Key Process Memory Infomation"
+	_placeTitle "Key Process Memory Infomation (MB)"
 	_placeLine "  Stats in MB and time" "Uptime\t\tPss\tShared\tRss\tVirtual"
-	_memperf "runner" "node /wigwag/devicejs-core-modules/Runner/start.js"
-	_memperf "devicejs-user" "=user"
+	#_memperf "runner" "node /wigwag/devicejs-core-modules/Runner/start.js"
```

Comment:
Any reason for keeping these around if we can already reference the commit history?